runtime/src/bank.rs

```diff
@@ -6792,12 +6794,13 @@ impl Bank {
         //  1. Transaction forwarding delay
         //  2. The slot at which the next leader will actually process the transaction
         // Drop the transaction if it will expire by the time the next node receives and processes it
-        let api = perf_libs::api();
-        let max_tx_fwd_delay = if api.is_none() {
-            MAX_TRANSACTION_FORWARDING_DELAY
-        } else {
-            MAX_TRANSACTION_FORWARDING_DELAY_GPU
-        };
+        // let api = perf_libs::api();
```

Comment:
Is `perf_libs` from the `solana-performance` crate?

Reply:
Yes, you are right